assets/src/design-system/components/input/index.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import propTypes from 'prop-types';
+import { useMemo } from 'react';
+import styled, { css } from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+/**
+ * Internal dependencies
+ */
+import { Text } from '..';
+import { themeHelpers, THEME_CONSTANTS } from '../../theme';
+
+const Container = styled.div`
+  margin: 0 12px;
+`;
+
+const Label = styled.label(
+  ({ disabled, theme }) =>
+    disabled &&
+    css`
+      ${Text} {
+        color: ${theme.colors.fg.disable};
+      }
+    `
+);
+
+const StyledInput = styled.input(
```

Comment:
Let's make sure width is getting addressed here in a manner that the editor panels can ingest

Reply:
The input now spans 100% width as a default!